apps/artist/templates/sections/overview.jade

```diff
@@ -14,20 +14,19 @@ mixin rail(title, viewAllUrl, section)
   .evenly-bisected-header
     - var artistMeta = viewHelpers.artistMeta(artist)
 
-    //- Temporarily suppress partner-submitted bios until they are
-    //- improved.
-    - var hasArtsyBlurb = viewHelpers.hasArtsyBlurb(artist)
     - var hasMeta = viewHelpers.hasOverviewHeaderMeta(artist)
     - var hasShows = statuses.shows
 
     .bisected-header-cell.js-artist-overview-header-left
       if hasMeta
         .artist-blurb.bisected-header-cell-section
           h2 Biography
-          if hasArtsyBlurb
+          if artist.biography_blurb && artist.biography_blurb.text
```

Comment:
This field will either be the Artsy bio, or the featured bio. So we can just always render it if present, plus the credit line (if present).